modules/serverless-jaeger-config.adoc

```diff
@@ -56,7 +56,7 @@ spec:
       debug: "false" <4>
 ----
```

Comment:
The zipkin-endpoint a few lines above must be enclosed in double quotes: `"http://jaeger-collector.default.svc.cluster.local:9411/api/v2/spans"`